src/diffusers/schedulers/scheduling_consistency_models.py

```diff
@@ -268,6 +268,7 @@ def step(
         sample: torch.FloatTensor,
         generator: Optional[torch.Generator] = None,
         return_dict: bool = True,
+        use_noise: bool = True
```

Comment:
I think it would be better to put the section of the function that computes the unnoised prediction in its own function that can be called. 

i.e.

```
def step(...):
    denoised = self.denoise(...)

    if len(self.timesteps) > 1:
         prev_sample = ...
    else:
        prev_sample = denoised

    return prev_sample

def denoise(...):
    ...
```

this would avoid extra global config to the class and the training script could just call denoise()

Reply:
That would also work for me - is probably indeed the better option (as my first proposal would force people to update the config after training, which is probs more brittle)